examples/twilio-text/README.md

```diff
@@ -37,11 +37,11 @@
 
     Edit `src/pages/index.js` to see your site update in real-time!
 
-3. **Deploy**
+4.  **Deploy**
```

Comment:
3.?

Reply:
no it was wrong before — open up the folded code, there's a 3 there.